Add ability to read other people's minds

n1313 opened the pull request:
![Mar-24-2024 13-58-10](https://github.com/FreeFeed/freefeed-react-client/assets/632081/5962a60c-af2c-490c-b405-e6cab45dadef)

This is a prototype and it probably needs a bit more work.

1. to see it in action, set your local date to 2024-04-01 or change `orbitDate` in config to today
2. there are no tests because the change is date-dependent (and I'm too lazy to write date-dependent tests). Tests will fail on 2024-04-01 :(

n1313 committed:
Add ability to read other people's minds

davidmz reviewed (changes requested):
Фича — 🔥🔥🔥 
Но есть одна небольшая проблема, напишу к конкретному месту в коде.

davidmz commented on src/components/orbit.jsx:
```diff
@@ -0,0 +1,63 @@
+      <div className={styles.children} ref={childrenRef}>
+        {children}
+      </div>
+      {childrenRef.current ? (
```
Вот тут, и ещё в строке 24 childrenRef.current используется прямо в рендере. Но при первом рендере он ещё не определён, он становится ненулевым только после него. В результате, при первом рендере иконки не летают, а начинают летать только когда компонент по какой-то причине перерендеривается. Это может случаться очень нескоро.

Поскольку это нужно только для чтения размера, то можно либо размер передавать в props (он всегда известен для юзерпика), либо читать его через useEffect, и перерендеривать Orbit сразу.

davidmz committed:
Pass size explicitly to the Orbit component

davidmz committed:
Use data-format instead of naïve toISOString

toISOString Date method always return datetime in UTC timezone with offset. So, the date part of this string is always in UTC.

davidmz commented:
Отправил пару изменений, посмотри?

n1313 commented:
По-моему, прекрасно. Спасибо!
